templates/counter.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Title</title>
+</head>
+<body>
+Counter: {{ count }}
```

Comment:
try to to base.html and inherit all templates from it; in the base template you will need to create blocks for title and content that you will overwrite in views templates